vllm/model_executor/neuron_model_loader.py

```diff
@@ -20,31 +25,88 @@
     torch.float32: "f32",
 }
 
+# Models supported by Neuron.
+_NEURON_SUPPORTED_MODELS = {
+    "LlamaForCausalLM": ("transformers_neuronx.llama.model",
+                         "LlamaForSampling", "LlamaForCausalLM"),
+    "MistralForCausalLM": ("transformers_neuronx.mistral.model",
+                           "LlamaForSampling", "MistralForCausalLM")
```

Comment:
```suggestion
                           "MistralForSampling", "MistralForCausalLM")
```
Mistral has own sampler in transformers_neuronx

Reply:
thanks for the great catch!